scripts/addVersion.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
if it's not meant to be directly executed I don't think it should be in the `scripts/` directory, because that's what that's for